### PR TITLE
Only require the parts of OCI that we need

### PR DESCRIPTION
**Pull request by agrare**
Requires:

- [x] https://github.com/ManageIQ/manageiq-providers-oracle_cloud/pull/52

Ensure the correct OCI packages are included. This is needed because we are no longer including all of the oci gem.

### CODE REVIEW

File: app/models/manageiq/providers/oracle_cloud/regions.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@ module ManageIQ
   module Providers::OracleCloud
     class Regions < ManageIQ::Providers::Regions
       private_class_method def self.from_source
-        require "oci"
+        require "oci/common"
```
Comment (agrare) [inline app/models/manageiq/providers/oracle_cloud/regions.rb:5]: `"oci/common"` only uses 7.2MiB and is enough to access the `OCI::Regions::REGION_ENUM` (NOTE `require "oci/regions"` does not work there is more setup needed before that can be accessed)

**Comment by agrare** [comment]
Requiring all of the classes which are needed for refresh uses only `61.8MiB`

**Comment by agrare** [comment]
This one is a little more involved so not marking `najdorf/yes?` at least for now

**Comment by kbrock** [comment]
I thought we were preferring `lazy_load` over `requires`. (sorry I'm in a "have a hammer, everything is a nail" mode)
Having said that, I don't know how we could use lazy load here.

**Comment by Fryguy** [comment]
Re: autoload see https://github.com/oracle/oci-ruby-sdk/pull/67

**Pull request closed without merging**